File: 프로그래머스/1/68644. 두 개 뽑아서 더하기/두 개 뽑아서 더하기.java
import java.util.ArrayList;
import java.util.List;

public class Solution {
	public int[] solution(int[] numbers) {
		List<Integer> sums = new ArrayList<>();

		for (int i = 0; i < numbers.length; i++) {
			for (int j = i+1; j < numbers.length; j++) {
				sums.add(numbers[i]+numbers[j]);
			}
		}

		return sums.stream().distinct().sorted().mapToInt(Integer::intValue).toArray();
	}
}

File: 프로그래머스/1/68644. 두 개 뽑아서 더하기/두 개 뽑아서 더하기.py
from typing import List


def solution(numbers: List[int]) -> List[int]:
    numbers_length = len(numbers)
    result = set()
    for i in range(numbers_length):
        for j in range(i+1, numbers_length):
            result.add(numbers[i] + numbers[j])
    return sorted(list(result))